skbeam/core/tests/test_correlation.py

```diff
@@ -279,13 +279,13 @@ def test_CrossCorrelator1d():
                                        np.nan, 2.275846e-14]))
 
     assert_array_almost_equal(ycorr_1D_symavg[::20],
-                              np.array([3.347542,  1.54268227,  0.86220476,
+                              np.array([-5.3002753,  1.54268227,  0.86220476,
```

Comment:
Oh, nevermind. By "the value was changed to back original value later" you mean you reverted the commit. Got it.